password

```diff
@@ -0,0 +1,50 @@
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:86:in `run'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:71:in `run'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:45:in `invoke'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/exe/rspec:4:in `<top (required)>'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/bin/rspec:22:in `load'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/bin/rspec:22:in `<main>'
+# 
+#   Showing full backtrace because every line was filtered out.
+#   See docs for RSpec::Configuration#backtrace_exclusion_patterns and
+#   RSpec::Configuration#backtrace_inclusion_patterns for more information.
+
+An error occurred while loading ./=.
+Failure/Error: __send__(method, file)
+
+LoadError:
+  cannot load such file -- /Users/nathanthomas/turing/3module_take2/projects/shredshare/=
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:2076:in `load'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:2076:in `load_file_handling_errors'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:1583:in `block in load_spec_files'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:1581:in `each'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:1581:in `load_spec_files'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:102:in `setup'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:86:in `run'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:71:in `run'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:45:in `invoke'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/rspec-core-3.9.0/exe/rspec:4:in `<top (required)>'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/bin/rspec:22:in `load'
+# /Users/nathanthomas/.rbenv/versions/2.4.1/bin/rspec:22:in `<main>'
+# 
+#   Showing full backtrace because every line was filtered out.
+#   See docs for RSpec::Configuration#backtrace_exclusion_patterns and
+#   RSpec::Configuration#backtrace_inclusion_patterns for more information.
+No examples found.
+
+
+Finished in 0.00006 seconds (files took 0.25206 seconds to load)
+0 examples, 0 failures, 2 errors occurred outside of examples
+
```

Comment:
I don't know what this one means. It kind of looks like just an error message put into a file?

Reply:
It does. Devise again. I spent some time cleaning up what devise creates for easier readability. I must have missed this one.